core/src/test/java/io/aiven/klaw/service/TopicControllerServiceTest.java

```diff
@@ -359,20 +647,35 @@ public void approveTopicRequestsFailure1() throws KlawException {
     when(clusterApiService.approveTopicRequests(
             anyString(), anyString(), anyInt(), anyString(), anyString(), any(), anyInt()))
         .thenReturn(new ResponseEntity<>(apiResponse, HttpStatus.OK));
-    when(manageDatabase.getTeamsAndAllowedEnvs(anyInt(), anyInt()))
-        .thenReturn(Collections.singletonList("1"));
+    when(commonUtilsService.getEnvsFromUserId(anyString()))
+        .thenReturn(new HashSet<>(Collections.singletonList("1")));
 
     ApiResponse apiResponse1 = topicControllerService.approveTopicRequests("" + topicId);
+    assertThat(apiResponse1.getResult()).isEqualTo("failure");
+  }
+
+  @Test
+  @Order(27)
+  public void approveTopicRequestsFailure2() throws KlawException {
```

Comment:
We should give it a proper name instead of `1` and `2`.

Reply:
Agree, updated.